samples/apps/copilot-chat-app/webapi/CopilotChatWebApi.csproj

```diff
@@ -23,6 +23,7 @@
     <PackageReference Include="Microsoft.Identity.Web" Version="2.11.0" />
     <PackageReference Include="PdfPig" Version="0.1.8-alpha-20230423-3898f" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
+    <PackageReference Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
since it's non-obvious as to why this is here, could you add a comment in the project file explaining?